pkg/app/run.go

```diff
@@ -655,6 +655,13 @@ func configureAuth(apiEndpoint string, args []string, f config.File, c api.HTTPC
 	if err != nil {
 		return nil, fmt.Errorf("failed to request OpenID Connect .well-known metadata (%s): %w", metadataEndpoint, err)
 	}
+	// Set a more meaningful error message when Fastly servers are unresponsive
+	// check if the response code is a 500 or above
+	if resp.StatusCode >= 500 {
```

Comment:
I'd prefer to use the stdlib constants here...

```suggestion
	if resp.StatusCode >= http.StatusInternalServerError {
```